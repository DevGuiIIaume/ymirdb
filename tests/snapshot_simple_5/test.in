set a 1
snapshot
set b 2
snapshot
set c 3
snapshot
set d 4
snapshot
set e 5
snapshot
list snapshots
drop 3
list snapshots
drop 1
drop 5
list snapshots
drop 2
drop 4
list snapshots
checkout 1
list entries
set f 6
snapshot
list snapshots
drop 6
set g 7
snapshot
checkout 7
list entries
bye
