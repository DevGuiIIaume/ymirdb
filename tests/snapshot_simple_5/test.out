> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> saved as snapshot 4

> ok

> saved as snapshot 5

> 5
4
3
2
1

> ok

> 5
4
2
1

> ok

> ok

> 4
2

> ok

> ok

> no snapshots

> no such snapshot

> e [5]
d [4]
c [3]
b [2]
a [1]

> ok

> saved as snapshot 6

> 6

> ok

> ok

> saved as snapshot 7

> ok

> g [7]
f [6]
e [5]
d [4]
c [3]
b [2]
a [1]

> bye
